content/applications/websites/livechat.rst

```diff
@@ -5,80 +5,287 @@
+   :align: center
+   :alt: View of the widget tab for Odoo Live Chat.
+
+Participate in a conversation
+=============================
+
+As explained above, *operators* are the users who will respond to live chat requests from customers.
+The information below outlines the necessary steps for operators participating in live chat
+conversations on an Odoo database.
+
+Set an online chat name
+-----------------------
+
+Before participating in a live chat, operators should update their *Online Chat Name*. This is the
+name that will be displayed to site visitors in the live chat conversation.
+
+To update the :guilabel:`Online Chat Name`, click on the user name in the upper-right corner of any
+page in the database. Select :guilabel:`My Profile` to open the :guilabel:`Profile` page. On the
+right side of the :guilabel:`Preferences` tab, locate the :guilabel:`Online Chat Name` field and
+enter the preferred name.
 
-Leave or join a channel
-=======================
+.. image:: livechat/my-profile.png
+   :align: center
+   :alt: View of the My Profile option in Odoo.
 
-Go to :menuselection:`Website --> Configuration --> Settings`, access your channel under
-*Live Chat*, and *Join Channel* or *Leave Channel*.
+If a users :guilabel:`Online Chat Name` is not set, the name displayed will default to the
+:guilabel:`User Name`.
 
-.. image:: livechat/joinchannel.png
-   :alt: View of a channel form and the option to join a channel for Odoo Live Chat
+.. example::
+   A user has their full name as their :guilabel:`User Name`, but they do not want to include their
+   last name in a live chat conversation. They would then set their :guilabel:`Online Chat Name` to
+   include only their first name.
+
+   .. image:: livechat/online-chat-name.png
+      :align: center
+      :alt: View of user profile in Odoo, emphasizing the Online Chat name field.
+
+Join or leave a channel
+-----------------------
+
+To join a live chat channel, go to the :menuselection:`Live Chat` app and click the :guilabel:`JOIN`
+button on the kanban card for the appropriate channel.
+
+Any channel where the user is currently active will show a :guilabel:`LEAVE` button. Click this
+button to disconnect from the channel.
+
+.. image:: livechat/leave-channel.png
+   :align: center
+   :alt: View of a channel form and the option to join a channel for Odoo Live Chat.
+
+.. important::
+   *Operators* that do not show any activity in Odoo for more than thirty minutes will be considered
+   disconnected, and subsequently removed from the channel.
+
+Manage live chat requests
+-------------------------
+
+When an operator is active in a channel, chat windows will open in the bottom right corner of the
+screen, no matter where they are in the database. They can participate in conversations without
+leaving their current location.
+
+.. tip::
+   Conversations can also be accessed by clicking the :guilabel:`Conversations` icon in the
+   :guilabel:`Menu Bar`.
```

Comment:
menu bar should be lowercase, and Im indecisive if it needs a guilabel - curious on @StraubCreative 's thoughts here

Reply:
I changed it to lowercase and removed the guilabel-because while yes it is part of the UI, it is location, not the actual part that the user is interacting with. @StraubCreative let me know your thoughts